### PR TITLE
Add Issue type to submit form

### PR DESCRIPTION
**Pull request by Simon-Busch**

### CODE REVIEW

**Comment by Simon-Busch** [comment]
This PR depends on https://github.com/code-423n4/api.code4rena.com/pull/625

File: src/components/reporter/SubmitFindings.tsx [hunk 1]
```diff
@@ -165,6 +185,45 @@ const SubmitFindings = ({
...
+        { uid: "kee5WSpUxLb", name: "Timing", color: "#A52A2A" },
+        { uid: "mDXa8cfPPxD", name: "Decimal", color: "#FF69B4" },
+        { uid: "FcYQv5CBiUA", name: "Upgradable", color: "#808080" },
+        { uid: "uRGDLdPNAts", name: "ERC4626", color: "#800080" },
+        { uid: "DmBWbW7i24W", name: "Governance", color: "#FF0000" },
+        { uid: "QPNNCv84XyA", name: "Access Control", color: "#FFFF00" },
+        { uid: "Ku7a94FLYkY", name: "Context", color: "#008000" },
+        { uid: "yrwmfCkkwni", name: "Error", color: "#008000" },
+        { uid: "xVV79K3nL9G", name: "Solmate", color: "#808080" },
+        { uid: "VAhTnhtYNGQ", name: "Library", color: "#A52A2A" },
```
Comment (hannahtuttle) [inline src/components/reporter/SubmitFindings.tsx:222]: This list/functions seems like we could add it to a utils file or a template or something like that, that way it's easier for people to update it and use in other places.

**Review by hannahtuttle — APPROVED** [review]

File: src/components/reporter/SubmitFindings.tsx [hunk 2]
```diff
@@ -165,6 +185,45 @@ const SubmitFindings = ({
     }
   }, [currentUser]);
 
+  const getIssueTypes = (): { issueTypes: any[] } => {
+    // Based on discussion with Nate
+    // Hardcode for now.
+    return {
```
Comment (natevw) [inline src/components/reporter/SubmitFindings.tsx:191]:
```suggestion
    return Promise.resolve({
```

File: src/components/reporter/SubmitFindings.tsx (hunk 1 above)
Comment (natevw) [reply src/components/reporter/SubmitFindings.tsx:222]: Yeah, this is meant to come from API but iiuc @geoffchan23 would like to have it for an upcoming contest in ~5 hours if possible.

The list will get updated occasionally but I think it's fine to hardcode for now and then we can just focus on getting the frontend UI for this contest.

**Review by natevw — COMMENTED** [review]
Looks workable as an MVP to me, but I'll let Mango do the official review!

**Comment by Simon-Busch** [comment]
Still need to add the possibility to edit this information

**Review by hannahtuttle — APPROVED** [review]
Tested this with submit and updated, it looks good and it's working.

**Pull request merged**